composer.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "neos/neos-development-collection",
+    "description": "Neos packages in a joined repository for pull requests.",
+    "type": "neos-package-collection",
+    "require": {
+        "typo3/flow": "*",
```

Comment:
This file is generated automatically. If it should be a version branch, then we would need to fix this in the package composer jsons inside.